screens/Authentication/index.tsx

```diff
@@ -62,8 +62,21 @@ export default function EntryScreen() {
             By clicking on
             <Text> Sign In / Create an account </Text>
             you agree to our{" "}
-            <Text style={styles.underline}>Terms of Services</Text>. Learn more
-            about our <Text style={styles.underline}>Privacy Policy</Text> and{" "}
+            <Text
+              onPress={() =>
+                navigation.navigate("StaticPage", { page: "terms" })
+              }
```

Comment:
can you create a reusable function for this which accepts the page value as a prop